lib/src/sai_redis_bridge.cpp

```diff
@@ -13,10 +13,12 @@ sai_status_t redis_get_bridge_stats(
     return SAI_STATUS_NOT_IMPLEMENTED;
 }
 
-sai_status_t redis_clear_bridge_stats(
+sai_status_t redis_get_bridge_stats_ext(
```

Comment:
i am not sure this needs to be modified.

Reply:
fixed